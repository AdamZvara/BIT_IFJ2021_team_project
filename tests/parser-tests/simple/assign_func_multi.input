require "ifj21"

function foo() : integer, integer, number, number
	return 5,8,5.8,6.1
end

function main()
    local a : integer
	local b : integer
	local c : number
	local d : number
    a,b,c,d = foo()
    write(a,b,c,d)
end

main()
