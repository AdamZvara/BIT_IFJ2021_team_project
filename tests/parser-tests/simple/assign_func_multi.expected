580x1.7333333333333p+20x1.8666666666666p+2